flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -295,6 +299,161 @@ private static void assertEquals(
     }
   }
 
+  public static void assertEqualsSafe(
```

Comment:
what does `Safe` mean here? should they just be called `assertEquals`?

nit: Iceberg coding style probably would name `recs` as full names `records`